src/ensembl/src/analyticsHelper.ts

```diff
@@ -2,12 +2,30 @@ export enum AnalyticsCategory {
   GLOBAL = 'Global'
 }
 
-export const buildAnalyticsObject = (category: string) => (data: {}) => ({
+export type AnalyticsOptions = {
+  action?: string;
+  label: string;
+  value?: string;
```

Comment:
Are we sure here? Can it be number/boolean/object?

Reply:
`value` has to be a number. 

Please refer the react-ga examples:
https://github.com/react-ga/react-ga#examples

Reply:
Thanks @msmanoj. I'll change the types according to that table.

Reply:
@msmanoj Will we be using `nonInteraction` and `transport` properties as well?